tests/db/model/generator/createWithCollectors.js

```diff
@@ -52,6 +52,8 @@ describe('tests/db/model/generator/createWithCollectors.js >', () => {
     .catch(done);
   });
 
+  // delete generator after each test
+  afterEach(() => tu.forceDelete(tu.db.Generator, new Date()));
```

Comment:
Are you sure you want to be sending new date here? I think that will delete generators created *after* that date.

Reply:
@annyhe